# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/IonBuilder.h
# Commit: fdbb1c215d78
# Full Hash: fdbb1c215d783a85c27d71ed8bb832690962c948
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2016-03-28 20:04:22
# Regressor Bug: 1165052
# File Overlap Count: 1
# Description:
#   Bug 1165052 - Part 4: Inline IsConstructor intrinsic. r=jandem
# ==============================================================================

diff -r 5e02cbf51e45 -r fdbb1c215d78 js/src/jit/IonBuilder.h
--- a/js/src/jit/IonBuilder.h	Wed Dec 23 12:46:35 2015 +0900
+++ b/js/src/jit/IonBuilder.h	Mon Nov 16 09:01:37 2015 +0900
@@ -907,6 +907,7 @@
 
     // Utility intrinsics.
     InliningStatus inlineIsCallable(CallInfo& callInfo);
+    InliningStatus inlineIsConstructor(CallInfo& callInfo);
     InliningStatus inlineIsObject(CallInfo& callInfo);
     InliningStatus inlineToObject(CallInfo& callInfo);
     InliningStatus inlineToInteger(CallInfo& callInfo);